### ISSUE TITLE
Any updates?

### ISSUE THREAD

**Issue by Johnny165** [machinamentum/CTR64#10]
Are there any updates?

**Comment by RandomMan25XX** [comment]
No, the whole thing is dead, no more chances of N64 emulation on the 3ds unless someone else tries their hand at it
